modules/reference/pages/properties.adoc

```diff
@@ -0,0 +1,6361 @@
+= Redpanda Configuration Properties 
+:description: Redpanda configuration properties. 
+
+== Broker
+
+Broker configuration properties are applied individually to each broker in a cluster. 
+
+IMPORTANT: After you change a broker-level property setting, you must restart the broker for the change to take effect. 
+
+---
+
+=== admin
+
+Address and port of admin server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/model::broker_endpoint
+
+*Default:* '{model::broker_endpoint(net::unresolved_address("127.0.0.1", 9644))}'
+
+=== admin_api_doc_dir
+
+Admin API doc directory
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* '/usr/share/redpanda/admin-api-doc'
+
+=== admin_api_tls
+
+TLS configuration for admin HTTP server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/endpoint_tls_config
+
+*Default:* None
+
+=== cloud_storage_cache_directory
+
+Directory for archival cache. Should be present when `cloud_storage_enabled` is present
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* None
+
+=== crash_loop_limit
+
+Maximum consecutive crashes (unclean shutdowns) allowed after which operator intervention is needed to startup the broker. Limit is not enforced in developer mode.
```

Comment:
```suggestion
Specifies the limit on the number of consecutive times a broker can crash within one hour before its crash-tracking logic is reset. This limit prevents a broker from getting stuck in an infinite cycle of crashes.

If null, the property is disabled and no limit is applied.

The crash-tracking logic is reset (to zero consecutive crashes) by any of the following conditions:

- The broker shuts down cleanly.

- One hour passes since the last crash.

- The broker configuration file, `redpanda.yaml`, is updated.

- The `startup_log` file in the broker’s [data_directory](https://docs.redpanda.com/current/reference/node-properties/#data_directory) is manually deleted.
```